docs/UserGuide.md

```diff
@@ -60,7 +60,7 @@ If you are looking to keep your physique, down to your finger muscles, in shape,
 
 ### Adding a client: `add client`
```

Comment:
Hi I know the add client was not your part cause I forgot about it :D but under the note, is it possible to add a point like
`* PHONE_NUMBER is unique to a client. Each phone number in the system must belong to exactly one client.`